content/ADR-151.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: adr
+adr: 151
+title: Rendering order for UI entities
+date: 2022-12-14
+status: Review
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - menduz
+---
+
+## Abstract
+
+This document describes the requirements to render the UI entities for the Decentraland Protocol. TL;DR; Pre-order using the entity tree defined by the UiTransform ([ADR-124](/adr/ADR-124)).
```

Comment:
Chat GPT's abstract hahahaha. It's just for fun.
```
This document describes a UI system in which entities are derived from a synthetic tree and rendered using pre-order sequencing. The rendering process allows for the configuration of stencils to control which parts of a parent entity are visible. The default behavior is to render children in full, even if they extend beyond the borders of the parent. This system is similar to how HTML, ReactNative, and Unity's UiToolkit handle rendering.
```

Reply:
Will use that abstract for the lulz and for the meta aspect of an AI defining the spec of Decentraland